README.md

```diff
@@ -34,21 +36,24 @@ Almost everything.
 - RelationPage: updating `availableToMentor` and `needsMentoring` status doesn't work
 - About screen is basically About dialog. It's very simple, more info should be added.
 
-Occasional UI bugs may occur – in this case, please create an issue.
+Occasional UI bugs may occur – in this case, please [file an issue](https://github.com/anitab-org/mentorship-flutter/issues/new/choose).
```

Comment:
Maybe you can add a Contributing section here and include a text to highlight the CONTRIBUTING guidelines that should be read before starting to contribute. Also the reporting guidelines and code of conduct